app/cp_nigeria/views.py

```diff
@@ -1354,8 +1339,11 @@ def get_pv_output(proj_id):
 @require_http_methods(["GET", "POST"])
 def get_community_details(request):
     community_id = request.GET.get("community_id")
-    community = Community.objects.get(pk=community_id)
-    data = {"name": community.name, "latitude": community.lat, "longitude": community.lon}
+    if len(community_id) == 0:
```

Comment:
```suggestion
    if community_id == "":
```

I prefer explicitly mentioning what we expect it to be